public/js/highlight-sequence.js

```diff
@@ -19,8 +20,38 @@ AFRAME.registerComponent("highlight-sequence", {
       } else {
         curElement.setAttribute("material", "color", "red");
       }
+      outlineElem(node);
+    };
+    const cleanupOutlines = () => {
+      const outlineElems = document.querySelectorAll("[isoutline='true']");
```

Comment:
didn't make `isoutline` camel case since the capitalization is automatically removed